doc/manual/src/command-ref/env-common.md

```diff
@@ -86,6 +86,11 @@ Most Nix commands interpret the following environment variables:
     from (defaults to the XDG spec locations). The variable is treated
     as a list separated by the `:` token.
 
+  - `NIX_OPTIONS`  
+    Applies settings from Nix configuration from the environment.
+    The content is treated as if it was read from a Nix configuration file.
+    Settings are separated by the `\n` token.
```

Comment:
```suggestion
    Settings are separated by the newline character.
```

Reply:
Otherwise it suggests you can separate them by writing an escaped "\n".

Reply:
Good point.